### PR TITLE
Flash programming is working¿¿¿¿

### PR DESCRIPTION
**Pull request by NicolaiFransen**
¿Por qué funciona?
No lo sé...

### CODE REVIEW

File: Software/Development/SemesterProject/System/systemInit.c [hunk 1]
```diff
@@ -134,6 +138,21 @@ void systemInit(void)
...
+    // symbols are created by the linker. Refer to the F2808.cmd file.
+    //
+    //memcpy(&RamfuncsRunStart, &RamfuncsLoadStart, (Uint32)&RamfuncsLoadSize);
+
+    //
+    // Call Flash Initialization to setup flash waitstates
+    // This function must reside in RAM
+    //
+    //InitFlash();
+
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/systemInit.c:147]: Why including this comments?

File: Software/Development/SemesterProject/System/Include/systemInit.h [hunk 2]
```diff
@@ -13,6 +13,7 @@
 //
 #include "../../App/Include/dutyCycle.h"
 #include "../../App/Include/openLoopControlManager.h"
+#include "string.h"
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/Include/systemInit.h:17]: Why?

**Review by AitorTeran — APPROVED** [review]
I have no idea what this is or even what it is for... but sure, if it works it works hahaha
I say ship it but you have to tell me what it's for

File: Software/Development/SemesterProject/System/systemInit.c (hunk 1 above)
Comment (NicolaiFransen) [reply Software/Development/SemesterProject/System/systemInit.c:147]: You will have to remove the comment when you want to flash the DSP.
A detailed description of what else to do is added to the trello card:
https://trello.com/c/XliY0e1j/116-watchdog-flash-programming

File: Software/Development/SemesterProject/System/Include/systemInit.h (hunk 2 above)
Comment (NicolaiFransen) [reply Software/Development/SemesterProject/System/Include/systemInit.h:17]: It's used for the memcpy()

**Review by murgui — APPROVED** [review]
I guess it's OK, I assume there's a nice description in OneNote about the procedure/use (not only in trello).

Ship it!

**Pull request merged**